features/support/helpers/search_helper.rb

```diff
@@ -19,5 +19,19 @@ def search_phrase(phrase)
       click_button("Search")
     end
 
+    def get_class_name(label)
+      class_hsh = {"Format" => "format_ssm", 
+                   "Date range" => "unitdate_ssm", 
+                   "Abstract" => "abstract_ssm", 
+                   "Library" => "repository_ssi", 
+                   "Call no" => "unitid_ssm",
+                   "Location" => "location_ssm",
+                   "Contained in" => "breadcrumb_ssm"
```

Comment:
@eshadatta You probably want to use the `Solrizer.solr_name` function to get these suffixes